components/pipeline-service/staging/stone-stg-m01/deploy.yaml

```diff
@@ -896,48 +896,6 @@ subjects:
   namespace: tekton-results
 ---
 apiVersion: v1
-data:
```

Comment:
our bypassing of defining config-logging in pipeline service in favor of the tekton-config based approach